src/libsync/filesystem.cpp

```diff
@@ -45,6 +46,16 @@ extern "C" {
 
 namespace OCC {
 
+QString FileSystem::longWinPath( const QString& inpath )
+{
+    QString path(inpath);
+
+    path.replace('/', '\\');
+    path.prepend(QLatin1String("\\\\?\\"));
```

Comment:
The csync version of this function has a special case for paths that start with / or \. Should this one have it too?